aredis_om/model/model.py

```diff
@@ -1267,6 +1271,29 @@ async def add(
 
         return models
 
+    @classmethod
+    async def delete_all(
+        cls,
+        models: Sequence["RedisModel"],
+        pipeline: Optional[Pipeline] = None,
+        pipeline_verifier: Callable[..., Any] = verify_pipeline_response,
+    ) -> int:
+        if pipeline is None:
```

Comment:
Any reason delete all shouldn't happen in a pipeline?    What happens when you have a massive amount of items to delete? Perhaps we should err on the side of bulking the deletes in groups of ~100 or so. Arguably, this could even become a configurable value.